setup.py

```diff
@@ -12,7 +12,7 @@
 
 dev_dependencies = [
     'flake8>=3.8,<4',
-    'isort>=4.3,<5',
+    'isort',
```

Comment:
you should lock the version to something, 5.xxx?

`isort>=5.somethingsomething,<6`

Reply:
otherwise we might have similar issues as 6 is released